src/html.pre.js

```diff
@@ -54,6 +68,7 @@ function pre(context) {
       });
     }
     $(document.body).children().wrapAll(div);
+    $sections = $(document.body).children('div');
```

Comment:
When there are no sections, we add a fake `div` wrapper, but didn't update the `sections` object.